src/CoreLib/GV.fs

```diff
@@ -1190,7 +1194,7 @@ type internal GVSerialize<'K>( ) =
     inherit GVSerialize( 
         let wrapperFunc (ms:MemStream) (O1:Object) =
             let state1 = O1 :?> 'K
-            ms.SerializeFrom( state1 )
+            Strm.SerializeFrom( ms, state1 )
```

Comment:
Is "Strm" a type name? What does it stands for?  In general, .Net framework guideline does not prefer using abbreviations for naming unless it's very well known (for example IO).   See:  https://msdn.microsoft.com/en-us/library/141e06ef(v=vs.71).aspx